azure_monitor/src/azure_monitor/sdk/auto_collection/live_metrics/__init__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+#
+import time
+import typing
+import uuid
+
+from azure_monitor.protocol import BaseObject, LiveMetricEnvelope
+from azure_monitor.utils import azure_monitor_context
+
+LIVE_METRICS_SUBSCRIBED_HEADER = "x-ms-qps-subscribed"
+STREAM_ID = str(uuid.uuid4())
+
+
+def create_metric_envelope(instrumentation_key: str):
```

Comment:
Can this logic be moved to `manager.py` so we don't need anything in `__init__,py`?

Reply:
Or to `exporter.py`

Reply:
Moved to utils.py to avoid circular dependencies